arangod/ClusterEngine/ClusterIndex.cpp

```diff
@@ -42,12 +42,15 @@ using Helper = arangodb::basics::VelocyPackHelper;
 namespace {
 /// @brief hard-coded vector of the index attributes
 /// note that the attribute names must be hard-coded here to avoid an init-order
-/// fiasco with StaticStrings::FromString etc.
+/// fiasco with StaticStrings::IdString etc.
+/// TODO FIXME
+constexpr std::string_view idAttribute("_id");
+constexpr std::string_view keyAttribute("_key");
```

Comment:
Are those values now present in the StaticStrings as string_views?

Reply:
No, they aren't yet.
I will not touch anything in `StaticStrings.cpp` for now, simply because the strings from it are used everywhere, even as lookup keys in containers. And I don't want to introduce additional overhead by potential std::string_view -> std::string conversions now, and would like to get rid of this already-too-big PR as soon as possible.
I think we can convert the strings in `StaticStrings.cpp` into string_views in a separate PR.